### PR TITLE
fix: only prefill if Integer versioning or greater than 1.0.0

### PR DESCRIPTION
**Pull request by cloeser**

### CODE REVIEW

File: app/lib/app/SecvisogramPage/View.js [hunk 1]
```diff
@@ -332,12 +335,18 @@ function View({
...
-        typeof tracking.version === 'string'
-          ? parseInt(tracking.version.split('.')[0])
-          : tracking.version
-      if (majorVersion >= 1) {
+      const semverVersion = semver.valid(tracking.version)
+      const initialPublicationVersion = semver.parse('1.0.0')
+
+      // prefill only if Integer versioning OR Semantic version is greater than 1.0.0 (after initial Publication)
+      if (
+        !semverVersion ||
```
Comment (valentindey) [inline app/lib/app/SecvisogramPage/View.js:345]: I'd find this a more clear naming:

```suggestion
      const isSemverVersion = semver.valid(tracking.version)
      const initialPublicationVersion = semver.parse('1.0.0')

      // prefill only if Integer versioning OR Semantic version is greater than 1.0.0 (after initial Publication)
      if (
        !isSemverVersion ||
```

File: app/lib/app/SecvisogramPage/View.js [hunk 2]
```diff
@@ -332,12 +335,18 @@ function View({
...
+        (semverVersion &&
+          initialPublicationVersion &&
```
Comment (valentindey) [inline app/lib/app/SecvisogramPage/View.js:347]: why do we need this check here?

File: app/lib/app/SecvisogramPage/View.js (hunk 1 above)
Comment (cloeser) [reply app/lib/app/SecvisogramPage/View.js:345]: Yes and No :)

semver.valid() is null if the version is no valid semver string and a semver object representing the version if it is. In the case of a valid semver string the naming would not match.

Comment (valentindey) [reply app/lib/app/SecvisogramPage/View.js:345]: What about `Boolean(semver.valid("tracking.version"))` ?

Comment (cloeser) [reply app/lib/app/SecvisogramPage/View.js:345]: We need the semverVersion object for comparison

Comment (valentindey) [reply app/lib/app/SecvisogramPage/View.js:345]: :see_no_evil: indeed

File: app/lib/app/SecvisogramPage/View.js (hunk 2 above)
Comment (cloeser) [reply app/lib/app/SecvisogramPage/View.js]: Good find, changed!
I was looking for this but did not find it in the documentation.

**Review by valentindey — APPROVED** [review]

**Pull request merged**